### DISCUSSION TITLE
Erro ! DocumentTotals.GrossTotal é maior que o documento de origem.

### DISCUSSION THREAD

**Discussion by Domingosantonio** [assoft-portugal/SAF-T-AO#218] [Q&A]
Alguém passando por isso?
Erro no ficheiro, AuditFile.SourceDocuments.SalesInvoices.2Invoice.References.Reference(FT FTC2022/6) ==== (NC NCC2022/4) .DocumentTotals.GrossTotal é maior que o documento de origem.
Como ultrapassar?

**Comment by gervasiogwoto** [comment]
Boa noite, também estou a me deparar com esse erro. porém já refiz os cálculos e não vejo aonde está a falha.
em pesquisas anterior dizem ser uma falha do Porta da AGT e devemos ignorar.
